pkg/network/usm/ebpf_ssl.go

```diff
@@ -525,6 +541,36 @@ func (o *sslProgram) ConfigureOptions(options *manager.Options) {
 
 // PreStart is called before the start of the provided eBPF manager.
 func (o *sslProgram) PreStart() error {
+	sslCtxByPIDTGIDMapObj, _, err := o.ebpfManager.GetMap(sslCtxByPIDTGIDMap)
+	if err != nil {
+		return fmt.Errorf("dead process ssl cleaner failed to get map: %q error: %w", sslCtxByPIDTGIDMap, err)
+	}
+
+	cleaner, err := ddebpf.NewMapCleaner[uint64, uint64](sslCtxByPIDTGIDMapObj, protocols.DefaultMapCleanerBatchSize/2, sslCtxByPIDTGIDMap, UsmTLSAttacherName)
+	if err != nil {
+		return fmt.Errorf("failed to create sslCtxByPIDTGIDMap cleaner: %w", err)
+	}
+	o.sslCtxByPIDTGIDMapCleaner = cleaner
+
+	sslSockByCtxMapObj, _, err := o.ebpfManager.GetMap(sslSockByCtxMap)
+	if err != nil {
+		return fmt.Errorf("dead process ssl cleaner failed to get map: %q error: %w", sslSockByCtxMap, err)
+	}
+	o.sslSockByCtxMapObj = sslSockByCtxMapObj
+	cleaner2, err := ddebpf.NewMapCleaner[uint64, http.SslSock](sslSockByCtxMapObj, protocols.DefaultMapCleanerBatchSize/2, sslSockByCtxMap, UsmTLSAttacherName)
+	o.sslSockByCtxMapCleaner = cleaner2
```

Comment:
missing `err` check here

Reply:
True, it was missed by us
thanks for reviewing